README.md

```diff
@@ -29,7 +36,7 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
+  <!-- - [🚀 Live Demo](#live-demo) -->
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
![image](https://github.com/unorjikingsley/Vet-Clinic-Database/assets/93039359/b4d7d17f-17cf-447a-9964-7203a7d06ed3)

- [ ] It would be nice if you fix the highlighted links in the image above because right now they are broken 😨 Kindly fix them or remove them (If you don't have any section for these link) to make your readme descriptive and professional 👍